app/assets/v2/js/pages/profile-trust.js

```diff
@@ -288,6 +288,46 @@ Vue.component('twitter-verify-modal', {
   }
 });
 
+Vue.component('google-verify-modal', {
+  delimiters: [ '[[', ']]' ],
+  data: function() {
+    return {
+      showValidation: false,
+      validationError: ''
+    };
+  },
+  mounted: function() {
+    $(document).on('click', '#verify-google-link', function(event) {
```

Comment:
Since this does not actually display a modal dialog, but simply executes a JavaScript function, I'm not sure we need to make a Vue component here. Seems like we should just include the `$(document).on('click')` in document.ready, and have the `requestyGoogleVerify` function as a freestanding function.

If/when we make the whole page one Vue app, we can include this functionality in a method, but for right now, creating a component with no actual UI seems like an anti-pattern.

Reply:
Removed it.